app/controllers/application_controller.rb

```diff
@@ -10,22 +10,23 @@ class ApplicationController < ActionController::Base
   unless Rails.application.config.consider_all_requests_local
     rescue_from ActiveRecord::RecordNotFound, :with => :record_not_found
     rescue_from ActionController::RoutingError, :with => :route_not_found
+    rescue_from ActionController::UnknownController, :with => :route_not_found
```

Comment:
thought i was handling an error for routing, but now that i look into it- it's a bit pointless.